### ISSUE TITLE
Fine-tuning the final layers of the detector

### ISSUE THREAD

**Issue by shani-sony** [OpenImagingLab/AdaptiveISP#4]
Is it possible to train the AdaptiveISP on a new dataset and fine-tun the final layers of the yolo-v3 detector?
I want to train the model on a dataset that has different classes than coco, in order to do that I changed the freezing code to not freeze the final layers but it appears to not work well (the results are very low), any idea what might be wrong? This is the code I changed:
train.py, L239
```
for k, v in self.yolo_model.named_parameters():
    if '28' not in k:
        v.requires_grad = False
```

**Comment by iimmortall** [comment]
Good idea, but maybe some additional training strategies are needed. For example, first, use a partial dataset with the same classes as COCO on the new dataset, train the AdaptiveISP, and freeze the YOLO-v3 detector. Second, freeze the AdaptiveISP and fine-tune only the final layers of the YOLO-v3 using the entire new dataset. Third, train both the AdaptiveISP and the YOLO-v3.

By the way, when fine-tuning the final layers of the YOLO-v3 detector, in addition to unfreezing the gradients, you should also include the final layer's parameters in the optimizer.

**Comment by shani-sony** [comment]
Thanks!
